crates/core/src/db/datastore/system_tables.rs

```diff
@@ -248,36 +260,13 @@ pub fn st_table_schema() -> TableSchema {
                 is_unique: true,
             },
         ],
-        columns: vec![
```

Comment:
I'm going to veto these changes. This is some of the most critical code in the whole system, and the changes here make it more difficult to read and less obvious what is really going on. Particularly with the field names not inline like they were.

Additionally this change seems unrelated to Sats, no?